core/src/smartcontracts/isi/tx.rs

```diff
@@ -66,7 +65,7 @@ impl ValidQuery for FindAllTransactions {
                 .flat_map(BlockTransactionIter::new)
                 .map(|tx| TransactionQueryOutput {
                     block_hash: tx.block_hash(),
-                    transaction: tx.value(),
+                    transaction: Box::new(tx.value()),
```

Comment:
why do we have to box it now?

Reply:
Because without expressions `Value` has become self-containing with unlimited size. `Value` contains `TransactionValue` which contains `Value`. So I had to put `Box` somewhere

Reply:
since there is no more expressions, I don't think we need `Value` in that way anymore. We can consider this in a separate issue

Reply:
- Created #4124